docs/cli-next/action.yml

```diff
@@ -20,25 +20,21 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - if: ${{ github.event_name == 'pull_request' }}
-      uses: ory/ci/checkout@master
-      with:
-        path: current-repo
-        fetch-depth: 0
-        ref: ${{ github.event.pull_request.head.ref }}
-        repository: ${{ github.event.pull_request.head.repo.full_name }}
-        token: ${{ inputs.token || github.token }}
```

Comment:
This means we checked out master instead of the PR's branch, but what is the point of that? Just not triggering this action in case it is a pull request seems more sensible.